parsigs/parse_sig_api.py

```diff
@@ -87,14 +88,27 @@ def _parse_sig(sig: str, model: Language):
     return _create_structured_sigs(model_output)
 
 
-def _pre_process(sig):
-    sig = sig.lower().replace('twice', '2 times').replace("once", '1 time').replace("nightly", "every night")
+def _autocorrect(sig):
+    spell = SpellChecker()
+    spell.word_frequency.load_text_file(os.path.join('data', 'drug_names.txt'))
```

Comment:
The spellchecker and file loading should be defined as global variables not to be loaded every call to the method

Regarding the added file- please add it to a directory called `resources`